corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/entities/menu.js

```diff
@@ -0,0 +1,112 @@
+/*global FormplayerFrontend */
+
+FormplayerFrontend.module("Entities", function (Entities, FormplayerFrontend, Backbone, Marionette, $) {
+
+    Entities.UserModel = Backbone.Model.extend({});
+
+    Entities.MenuSelect = Backbone.Model.extend({
+        urlRoot: "menuSelects",
+    });
+
+    Entities.MenuSelectCollection = Backbone.Collection.extend({
+
+        model: Entities.MenuSelect,
+
+        parse: function (response) {
```

Comment:
so based on the response this can be a collection of two different types of models?